src/components/PopupBase/__tests__/PopupBase.spec.tsx

```diff
@@ -14,28 +16,32 @@ describe('<PopupBase> that adds basic anchor/popup functionality to rendered com
     const popupRendererMock = ({ setPopupVisibility }) => (
         <PopoverDummy setPopupVisibility={setPopupVisibility} />
     );
-    let wrapper;
+    let view: RenderResult;
+    let user;
 
     describe('rendering', () => {
         beforeEach(() => {
-            wrapper = mount(
+            view = render(
                 <PopupBase anchorRenderer={anchorRendererMock} popupRenderer={popupRendererMock} />
             );
+            user = userEvent.setup();
         });
 
         it('should render with minimal props correctly', () => {
-            expect(toJson(wrapper)).toMatchSnapshot();
-            expect(wrapper.find('Popover')).toHaveLength(0);
+            expect(view.container).toMatchSnapshot();
```

Comment:
I think this is not also producing the expected snapshot. Look at the .snap file.

Reply:
I was original snapshot
![image](https://github.com/textkernel/oneui/assets/104063210/23a8fa52-6d44-4e43-b904-2369359f99db)

Reply:
This new snapshot
![image](https://github.com/textkernel/oneui/assets/104063210/f7377bf8-5dde-463b-a814-7e2e72fc25c1)

Reply:
This one will be if use expect(view.baseElement).toMatchSnapshot()
![Uploading image.png…]()

Reply:
Sorry, you're right, I think I mentioned the wrong test.